CMakeLists.txt

```diff
@@ -22,16 +22,20 @@ set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
 add_subdirectory("${CMAKE_BINARY_DIR}/googletest-src"
                  "${CMAKE_BINARY_DIR}/googletest-build")
 
-# find dependencies
+# find dependencies (MPI, Kokkos, and HYPRE found through Cabana)
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
 find_package(CLANG_FORMAT 10)
-find_package(MPI REQUIRED)
-find_package(Kokkos REQUIRED)
-find_package(Cabana REQUIRED COMPONENTS Cabana::Cajita Cabana::cabanacore)
-find_package(HYPRE REQUIRED)
```

Comment:
So the issue was we were already looking for these?

Reply:
We were looking for hypre here, but not linking in `src/CMakeLists.txt`